apps/src/code-studio/clientState.js

```diff
@@ -159,33 +140,6 @@ function levelProgressByScript() {
   }
 }
 
-/**
- * Returns the number of lines completed from the cookie.
- *
- * TODO: The user's total line count is no longer shown anywhere in the UI.
- * Remove this as part of LP-2291 to clean up the code that stores and
- * maintains the total line count.
- *
- * @returns {number}
- */
-clientState.lines = function() {
-  var linesStr = sessionStorage.getItem('lines');
-  return isFinite(linesStr) ? Number(linesStr) : 0;
-};
-
-/**
- * Adds the given number of completed lines.
- * @param {number} addedLines
- */
-function addLines(addedLines) {
-  var newLines = Math.min(
-    clientState.lines() + Math.max(addedLines, 0),
-    MAX_LINES_TO_SAVE
```

Comment:
I'm curious why this logic was here? Why couldn't we add 2000 lines to the total? (You might not know the answer but I'm very curious...)

Reply:
I found a very very old PR about this.... https://github.com/code-dot-org/code-dot-org/pull/641

Reply:
lolol and the title of the PR pretty much says it all "don't allow people to send us nonsense numbers to mess up the lines of code counter"